devops/docker/README.md

```diff
@@ -0,0 +1,40 @@
+## Docker Image Release Workflow
+
+To login to dockerhub:
+```bash
+docker login
+```
+
+To build image:
+```bash
+docker build -t <org>/<image-name>:<version> .
+```
+
+Where `version` is <jdk-version>-<idea-version>.
```

Comment:
>idea-version

this is too specific for one docker container ... I thought you tried to make general doc for all.
general versioning is `-` separate versions of packges that is in docker 
example https://hub.docker.com/r/checkstyle/jdk-11-groovy-git-mvn/tags
or https://hub.docker.com/r/checkstyle/idea-docker/tags

Reply:
done